util/localconfig/file_perm_windows.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 func getFilePermission(fi os.FileInfo) error {
-	if fi.Mode().Perm() == 0666 || fi.Mode().Perm() == 0444 {
+	if fi.Mode().Perm() == 0o666 || fi.Mode().Perm() == 0o444 {
```

Comment:
I think we should focus only on the error handling in this PR, let's add this change separately.